lib/logstash/outputs/elasticsearch.rb

```diff
@@ -62,6 +62,17 @@
 #
 # Keep in mind that a connection with keepalive enabled will
 # not reevaluate its DNS value while the keepalive is in effect.
+#
+# ==== HTTP Compression
+#
+# This plugin supports request and response compression. Response compression is enabled by default and 
+# for Elasticsearch versions 5.0 and later, the user doesn't have to set any configs in Elasticsearch for 
+# it to send back compressed response. For versions before 5.0, `http.compression` must be set to `true` in 
```

Comment:
also, should we reference the docs here? https://www.elastic.co/guide/en/elasticsearch/reference/5.0/modules-http.html#_settings_2